README.md

```diff
@@ -74,6 +74,7 @@ There are multiple **blockchain backends** available. A backend is automatically
 - **Ethereum.** The Ethereum backend is available at [perun-eth-backend](https://github.com/perun-network/perun-eth-backend/).
 - **Polkadot.** The Polkadot backend is available at [perun-polkadot-backend](https://github.com/perun-network/perun-polkadot-backend).
 - **Cosmos.** The Cosmos backend is available at [perun-cosmwasm-backend](https://github.com/hyperledger-labs/perun-cosmwasm-backend).
+- **Fabric** The Hyperledger Fabric backend is available at [perun-fabric](https://github.com/perun-network/perun-fabric).
```

Comment:
Should we rename to `perun-fabric-backend`? ;)

Reply:
I had the same thought 😂